httpd/handler.go

```diff
@@ -175,6 +179,127 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 	httpResults(w, results, pretty)
 }
 
+type Point struct {
```

Comment:
`omitempty` will stop any fields that don't have data, so they would be the same size.  Again, just a talking point, but wanted to point out that `omitempty` is pretty amazing.  You might want to put it on any fields in your struct that could be empty as well.